### PR TITLE
feat(pagination): create pagination component

### PR DESCRIPTION
**Pull request by marinasmah**
## Description

Create new component "Pagination", according to Spec

## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by boiarchukk — DISMISSED** [review]

File: web-components/src/components/pagination/Pagination.stories.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  return html`
+    <md-pagination total-page="20" current-page="10" ?no-navigation=${navigation}></md-pagination>
+  `;
+};
+export const onlyDots = () => {
+  const onlyDots = boolean("Only Dots", true);
+  return html`
+    <md-pagination total-page="20" current-page="10" ?only-dots=${onlyDots}></md-pagination>
+  `;
+};
```
Comment (futurethang) [inline web-components/src/components/pagination/Pagination.stories.ts:48]: Nit: we can make this reflect the single-output pattern that Helen refactored just recently so all the knobs are on one page

File: web-components/src/components/pagination/tokens/lm-pagination-tokens.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    dark: colors.gray[10].name
+  },
+  nav: {
+    color: {
+      light: colors.gray[90].name,
+      dark: colors.gray[40].name
+    },
+    focus: {
+      light: colors.blue[60].name,
+      dark: colors.blue[40].name
```
Comment (futurethang) [inline web-components/src/components/pagination/tokens/lm-pagination-tokens.js:32]: available as global CSS var `--md-default-focus-outline-color` , no need to duplicate

File: web-components/src/components/pagination/tokens/lm-pagination-tokens.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    },
+    dark: colors.gray[40].name,
```
Comment (futurethang) [inline web-components/src/components/pagination/tokens/lm-pagination-tokens.js:34]: Where is this used?

**Review by futurethang — CHANGES_REQUESTED** [review]

File: web-components/src/components/pagination/Pagination.stories.ts (hunk 1 above)
Comment (marinasmah) [reply web-components/src/components/pagination/Pagination.stories.ts]: Fixed

File: web-components/src/components/pagination/tokens/lm-pagination-tokens.js (hunk 3 above)
Comment (marinasmah) [reply web-components/src/components/pagination/tokens/lm-pagination-tokens.js]: Fixed

**Comment by yananym** [comment]
@marinasmah please remember to include this component to the list of exported components in index.ts file - otherwise library consumers are not able to access it

**Review by futurethang — APPROVED** [review]

**Review by elenagiere — APPROVED** [review]

**Pull request merged**